app/src/features/surveys/view/SurveyObservations.tsx

```diff
@@ -289,72 +313,183 @@ const SurveyObservations: React.FC = () => {
         <Divider />
 
         <Box p={3}>
-          {!submissionExists ? (
-            <>
-              <Box textAlign="center">
-                <Typography data-testid="observations-nodata" variant="body2" color="textSecondary">
-                  No Observation Data. &nbsp;
-                  <Link onClick={handleOpenImportObservations}>Click Here to Import</Link>
-                </Typography>
-              </Box>
-            </>
-          ) : (
+          {!occurrenceSubmission?.isValidating && (
             <>
-              <Alert
-                icon={<Icon path={submissionStatusIcon} size={1} />}
-                severity={submissionStatusSeverity}
-                action={submissionAlertAction()}>
-                <Box display="flex" alignItems="center" m={0}>
-                  <Link className={classes.alertLink} component="button" variant="body2" onClick={openFileContents}>
-                    <strong>{occurrenceSubmission?.inputFileName}</strong>
-                  </Link>
-                </Box>
-                <Typography variant="body2">
-                  {occurrenceSubmission?.isValidating
-                    ? 'Validating observation data. Please wait...'
-                    : occurrenceSubmission?.status}
-                </Typography>
-              </Alert>
-
-              {!occurrenceSubmission?.isValidating && (
-                <>
-                  {submissionStatusSeverity === 'error' && (
-                    <Box mb={2} mt={3}>
-                      <Typography data-testid="observations-error-details" variant="body1">
-                        Resolve the following errors in your local file and re-import.
-                      </Typography>
-                    </Box>
-                  )}
-
-                  {submissionMessageTypes.length > 0 && (
-                    <Box mt={1}>
-                      {
-                        // Alphabetize message types for consistency
-                        submissionMessageTypes.map((messageType) => {
-                          return (
-                            <Box key={messageType.messageTypeLabel}>
-                              <Alert severity={alertSeverityFromSeverityLabel(messageType.severityLabel)}>
-                                {messageType.messageTypeLabel}
-                              </Alert>
-                              <Box component="ul" my={3}>
+              {submissionStatusSeverity === 'error' && (
+                <Box mb={3}>
+                  <Alert severity="error" icon={<Icon path={mdiAlertCircleOutline} size={1} />}>
+                    <AlertTitle>Failed to import observations</AlertTitle>
+                    One or more errors occurred while attempting to import your observations file.
+                    {
+                      // Alphabetize message types for consistency
```

Comment:
This comment suggests the message types are sorted; is that actually the case?

Reply:
removed